src/main/java/io/sqooba/traildb/TrailDBEvent.java

```diff
@@ -59,18 +59,28 @@ public long getNumItems() {
 
     /**
      * Get the decoded value of an item.
-     * 
+     *
      * @param index The item index.
      * @return The decoded item as a String.
      */
     public String getFieldValue(int index) {
         // Caching possibility here to avoid going back to JNI to decode items.
-        return this.trailDB.getItemValue(items[index]);
+        final ByteBuffer bb = ByteBuffer.allocate(8);
```

Comment:
use a constant for this magic number (8)

Reply:
I plan to remove all these ByteBuffer soon (: